.pullapprove.yml

```diff
@@ -0,0 +1,23 @@
+approve_by_comment: true
+approve_regex: ^Approved
+reject_regex: ^Rejected
+reset_on_push: true
+author_approval: required
+
+reviewers:
+  -
+      name: admins
+      required: 1
+      members:
+        - fred333
```

Comment:
Yea we should probably just used Github teams. If we hash those out I'll use those instead. That ways we don't have to worry about plain-text names and updating this file when people move around.